src/main/scala/nl/knaw/dans/easy/stage/dataset/Util.scala

```diff
@@ -67,6 +67,24 @@ object Util {
     } yield audience.text
   }
 
+  def readFileType(filePath: String)(implicit s: Settings): Try[Option[String]] = Try {
+    val filetype = for {
+      file <- loadBagXML("metadata/files.xml") \\ "files" \ "file"
+      if (file \@ "filepath") == filePath
+      filetype <- file \ "type"
+    } yield filetype
+    if (filetype.size == 1) Option(filetype.head.text)
+    else None
+  }
+
+  def isAVType(filePath: String)(implicit s: Settings): Try[Boolean] = {
+    readFileType(filePath).flatMap {
```

Comment:
simplify this to a `map`: you return a `Success(_)` in all cases, so a `flatMap` is not needed.